.changesets/maint_garypen_4025_helm_docs.md

```diff
@@ -0,0 +1,9 @@
+### Use a more modern golang with helm-docs ([Issue #4025](https://github.com/apollographql/router/issues/4025))
+
+Recent updates to helm-docs mean that the golang on the Operating System is too old.
+
+Download a version from the golang website and use that.
+
+fixes: #4025 
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/4027
```

Comment:
Let's remove this changelog entry. It's not user actionable.

Reply:
Isn't that the case for most chores? I thought we had the maintenance section to let people know what we'd been doing.

Reply:
At least it seems to be the trend right now to not include these.

Will add a suggestion if we're going to retain it.

Reply:
I removed it.